tasks/Task.hpp

```diff
@@ -4,6 +4,20 @@
 #define VISP_TASK_TASK_HPP
 
 #include "visp/TaskBase.hpp"
+#include <visp/vpConfig.h>
```

Comment:
Don't include things in the headers that you don't use in the headers. Try to move as many headers as possible into the .cpp file (e.g. I'm pretty sure that vpAdaptiveGain could be moved there)